modules/openshift-developer-cli-sample-applications.adoc

```diff
@@ -0,0 +1,114 @@
+[id='odo-sample-applications']
+= Sample applications
+include::openshift-developer-cli-attributes.adoc[]
+
+:context: odo-sample-applications
+
+{odo-title} is compatible with any language or run time listed within the OpenShift catalog of component types. In order to access the component over the web, you can create a URL using `odo url create`.
+
+Example:
```

Comment:
I'd remove this line and either make a title heading on the source block, or just add a "For example:" at the end of the above paragraph.